meta-resin-common/recipes-support/resin-vars/resin-vars/resin-vars

```diff
@@ -34,9 +34,17 @@ while [[ $# > 0 ]]; do
     shift
 done
 
+# Retrive resin-boot mountpoint
+BOOT_MOUNTPOINT="$(grep $(readlink -f /dev/disk/by-label/resin-boot) /proc/mounts | cut -d ' ' -f 2)"
+
+if [ -z "$BOOT_MOUNTPOINT" ]; then
+    echo "[ERROR] : Could not determine mount point for resin-boot partition"
+    exit 1
```

Comment:
There is no way CONFIG_PATH can reach a state where it is not set. It will at least get the default value (next if statement). This if statement should never yield true, but it is here just in case (code resilience)